src/QsCompiler/TestProjects/test18/Bell.qs

```diff
@@ -1,11 +1,19 @@
-﻿open Microsoft.Quantum.Intrinsic;
-open Microsoft.Quantum.Canon;
-
-@EntryPoint()
```

Comment:
This is kind of awkward. Because the stand-in project doesn't have references for these standard library namespaces, we get a bunch of diagnostics for that. That wasn't the point of that test (it's about namespace blocks), so I'm commenting this until we add the references soon